consumers/writers/influxdb/consumer_test.go

```diff
@@ -118,107 +118,103 @@ func queryDB(fluxQuery string) (int, error) {
 }
 
 func TestSaveSenml(t *testing.T) {
-	for i := 0; i < 2; i++ {
-		// Testing both async and sync
-		repo := writer.New(client, repoCfg, i == 0)
-
-		cases := []struct {
-			desc         string
-			msgsNum      int
-			expectedSize int
-		}{
-			{
-				desc:         "save a single message",
-				msgsNum:      1,
-				expectedSize: 1,
-			},
-			{
-				desc:         "save a batch of messages",
-				msgsNum:      streamsSize,
-				expectedSize: streamsSize,
-			},
-		}
+	// Testing both async and sync
```

Comment:
You can separate Async and Sync cases, it's a little cleaner solution. The same goes for JSON and Readers tests.